robot_stack/RobotJointAngles Joint_Angles_N
bool service_flag
---
geometry_msgs/Transform End_Effector_Transformation